### ISSUE TITLE
valid-types rule complains about @requires

### ISSUE THREAD

**Issue by adp-psych** [gajus/eslint-plugin-jsdoc#840]
I have this at the top of a file:

```js
/**
 * @file The scenarios of the experiment.
 *
 * @module module:scenarios/scenarios
 * @requires scenarios/effects
 * @author Anthony Di Pietro <anthony.dipietro@research.uwa.edu.au>
 * @copyright © 2020, 2021 Anthony Di Pietro
 * @license AGPL-3.0-or-later
 */
```

When I run ESLint, I get the following warning:

```
/home/user/.projects/psychology-phd/experiments/3/src/scenarios/scenarios.js
  22:0  warning  Syntax error in namepath: scenarios/effects  jsdoc/valid-types
```

I don’t understand why I’m getting this warning, and I wonder whether it might be spurious.

What should I do about this?

**Comment by brettz9** [comment]
Thanks for the report. In cases where `module:` was omitted, we had not been supplying our types parser with a copy that did have the `module:` path (so it could confirm the path was valid). Now fixed.
